buildSrc/buildSrc/src/main/kotlin/Deps.kt

```diff
@@ -0,0 +1,67 @@
+object Deps {
+
+    private const val kotlinVersion = "1.3.50"
+    private const val coroutinesVersion = "1.3.2"
+
+    val kotlin = Kotlin
+    val kotlinx = Kotlinx
+    val android = Android
+    val rxjava2 = "io.reactivex.rxjava2:rxjava:2.2.7"
+    val rxjava3 = "io.reactivex.rxjava3:rxjava:3.0.0-RC3"
+    val picasso = "com.squareup.picasso:picasso:2.71828"
+    val bintray = "com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.4"
+    val detekt = "io.gitlab.arturbosch.detekt:detekt-gradle-plugin:1.1.0"
+
+    object Kotlin {
+        val stdlib = Stdlib()
+        val test = Test
+        val plugin = "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
+
+        class Stdlib(
+            private val name: String = "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
+        ) : CharSequence by name {
+            val jdk7 = "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
+            val js = "org.jetbrains.kotlin:kotlin-stdlib-js:$kotlinVersion"
+            val common = "org.jetbrains.kotlin:kotlin-stdlib-common:$kotlinVersion"
+
+            override fun toString(): String = name
+        }
+
+        object Test {
+            const val common = "org.jetbrains.kotlin:kotlin-test-common:$kotlinVersion"
+            const val js = "org.jetbrains.kotlin:kotlin-test-js:$kotlinVersion"
+            const val junit = "org.jetbrains.kotlin:kotlin-test-junit:$kotlinVersion"
+            const val annotationsCommon = "org.jetbrains.kotlin:kotlin-test-annotations-common:$kotlinVersion"
+        }
+    }
+
+    object Kotlinx {
+        val coroutines = Coroutines
```

Comment:
Why do we need such fields? Why not to reference object directly? Like `Deps.Kotlinx.Coroutines.blah`. Currently you can use both ways btw.

Reply:
Can we make them objects everywhere so they will be like groups?

Reply:
> Why do we need such fields?

You can't use `Deps.Kotlinx.Coroutines.blah` in Groovy. Only via `Deps$Kotlinx`.

Reply:
> Can we make them objects everywhere so they will be like groups?

Did not get it. Sample?